skimage/data/_registry.py

```diff
@@ -0,0 +1,142 @@
+# flake8: noqa
+
+# This minimal dataset was available as part of
+# scikit-image 0.15 and will be retained until
+# further notice.
+# Testing data and additional datasets should only
+# be made available by pooch
```

Comment:
What does this mean exactly? ie how does one make a dataset available via pooch?